src/uct/sm/mm/posix/mm_posix.c

```diff
@@ -64,20 +65,28 @@ typedef struct uct_posix_packed_rkey {
 
 
 static ucs_config_field_t uct_posix_md_config_table[] = {
-  {"MM_", "", NULL,
-   ucs_offsetof(uct_posix_md_config_t, super), UCS_CONFIG_TYPE_TABLE(uct_mm_md_config_table)},
-
-  {"DIR", UCT_POSIX_SHM_OPEN_DIR,
-   "The path to the backing file. If it's equal to " UCT_POSIX_SHM_OPEN_DIR " then \n"
-   "shm_open() is used. Otherwise, open() is used.",
-   ucs_offsetof(uct_posix_md_config_t, dir), UCS_CONFIG_TYPE_STRING},
-
-  {"USE_PROC_LINK", "y", "Use /proc/<pid>/fd/<fd> to share posix file.\n"
-   " y   - Use /proc/<pid>/fd/<fd> to share posix file.\n"
-   " n   - Use original file path to share posix file.\n",
-   ucs_offsetof(uct_posix_md_config_t, use_proc_link), UCS_CONFIG_TYPE_BOOL},
-
-  {NULL}
+    {"MM_", "", NULL, ucs_offsetof(uct_posix_md_config_t, super),
+     UCS_CONFIG_TYPE_TABLE(uct_mm_md_config_table)},
+
+    {"DIR", UCT_POSIX_SHM_OPEN_DIR,
+     "The path to the backing file. If it's equal to " UCT_POSIX_SHM_OPEN_DIR
+     " then \n"
+     "shm_open() is used. Otherwise, open() is used.",
+     ucs_offsetof(uct_posix_md_config_t, dir), UCS_CONFIG_TYPE_STRING},
+    {"SHM_MIN_SIZE", "16mb",
+     "This is the minimum size of the backing file that enforced by UCX.\n"
+     "Default value ensures sufficient resources to most use cases.\n"
+     "Limiting memory space can lead to undefined behavior of UCX.",
```

Comment:
Minimal size of the shared memory file system. If the file system size is less than this value, the transport will be disabled since it may not be able to allocate sufficient resources for communications.